src/SFA.DAS.EmploymentCheck.Functions/ServiceCollectionExtensions.cs

```diff
@@ -22,25 +30,60 @@ public static class ServiceCollectionExtensions
         public static IServiceCollection AddEmploymentCheckService(this IServiceCollection serviceCollection, string environmentName)
         {
             serviceCollection.AddHttpClient();
-            serviceCollection.AddTransient<IAccountsApiClient, AccountsApiClient>();
-            serviceCollection.AddTransient<IAccountsService, AccountsService>();
-            serviceCollection.AddTransient<IHmrcService, HmrcService>();
+            serviceCollection.AddTransient<IEmploymentCheckClient, EmploymentCheckClient>();
+            serviceCollection.AddTransient<IEmployerAccountClient, EmployerAccountClient>();
+            serviceCollection.AddTransient<ISubmitLearnerDataClient, SubmitLearnerDataClient>();
+            serviceCollection.AddTransient<IEmployerAccountApiClient, EmployerAccountApiClient>();
+            serviceCollection.AddTransient<IHmrcClient, HmrcClient>();
+
+            // #if DEBUG
+            // For local development use the Stubs
+
+           // //serviceCollection.AddTransient<IEmploymentCheckService, EmploymentCheckServiceStub>();
+           // serviceCollection.AddTransient<IEmploymentCheckService, EmploymentCheckServiceStub>();
+           // serviceCollection.AddTransient<ISubmitLearnerDataService, SubmitLearnerDataServiceStub>();
+           //// serviceCollection.AddTransient<IEmployerAccountService, EmployerAccountServiceStub>();
+            //serviceCollection.AddTransient<IHmrcService, HmrcServiceStub>();
```

Comment:
Probably needs deleting now, especially as the #if DEBUG statement doesn't have a matching #endif

Reply:
agreed & deleted